docs/src/FAQ.md

```diff
@@ -76,3 +76,17 @@ ChainRulesTestUtils.jl has some dependencies, so it is a separate package from C
 This means your package can depend on the light-weight ChainRulesCore.jl, and make ChainRulesTestUtils.jl a test-only dependency.
 
 Remember to read the section on [On writing good `rrule` / `frule` methods](@ref).
+
+## How do chain rules work for complex functions?
+
+`ChainRules.jl` follows the convention that `frule` applied to a function `f(x + im*y) = u(x,y) + im*v(x,y)` with perturbation `ẋ + im*ẏ` returns
+```
+∂u/∂x * ẋ + ∂u/∂y * ẏ + im * ( ∂v/∂x * ẋ    ∂v/∂y * ẏ )
+```
+and similarly `rrule` applied to the same function with adjoint `ū + im*v̄` returns
+```
+ū * ∂u/∂x + v̄ * ∂v/∂x + im * ( ū * ∂u/∂y + v̄ * ∂v/∂y )
+```
+Note that these rules can be interpreted as multiplication with the Jacobian and transposed Jacobian, respectively of `(x,y)->[u(x,y), v(x,y])`. 
```

Comment:
@ettersi could you provide an expression for this interpretation directly please? i.e. write out the jvp and vjp in matrix-vector form?

Reply:
Sure. How do I insert a display math equation?

Reply:
Done